evm_loader/cli/src/main.rs

```diff
@@ -1033,6 +1033,29 @@ fn command_get_ether_account_data (
     }
 }
 
+fn command_neon_elf(
+    _config: &Config,
+    program_location: &str,
+) {
+    let program_data = read_program_data(program_location).unwrap();
+    let program_data = &program_data[..];
+    let elf = goblin::elf::Elf::parse(program_data).expect("Unable to parse ELF file");
+    elf.dynsyms.iter().for_each(|sym| {
+        let name = String::from(&elf.dynstrtab[sym.st_name]);
+        if name.starts_with("NEON")
+        {
+            let end = program_data.len();
+            let from = usize::try_from(sym.st_value).unwrap_or_else(|_| panic!("Unable to cast usize from u64:{:?}", sym.st_value));
+            let to = usize::try_from(sym.st_value + sym.st_size).unwrap_or_else(|err| panic!("Unable to cast usize from u64:{:?}. Error: {}", sym.st_value + sym.st_size, err));
+            if to < end && from < end {
```

Comment:
What happened if the parameter is outside the end? Maybe we need to write a warning on `stderr`?

Reply:
We just won't read it

Reply:
I suppose it's worth reporting that we were unable to retrieve a parameter. Then it will be easier to understand why some value was not included in the output if we encounter such a situation.

Reply:
Fixed